config/crd/bases/starlingx.windriver.com_hostprofiles.yaml

```diff
@@ -199,6 +199,10 @@ spec:
                 description: Console defines the installation output device.
                 pattern: ^(|tty[0-9]+|ttyS[0-9]+(,\d+([a-zA-Z0-9]+)?)?|ttyUSB[0-9]+(,\d+([a-zA-Z0-9]+))?|lp[0-9]+)$
                 type: string
+              hwSettle:
```

Comment:
Seem this actual CRD is not updated after adding the validation, need to re-generate the CRD.

Reply:
Re-generated CRD for update after adding the validation.